installscripts/jazz-terraform-unix-noinstances/scripts/scmpush.sh

```diff
@@ -12,14 +12,30 @@ ns_id_slf=$6
 # Config variables
 scm=$7
 jenkins_elb=$8
-jenkins_user=$9
-jenkins_password=${10}
-jazzbuildmodule=${11}
-gitlab_trigger_job_url="/project/Gitlab-Trigger-Job"
-jazz_ui_trigger_job_url="/project/jazz_ui"
+jazzbuildmodule=$9
+jenkins_user="jobexec"
+jenkins_password="jenkinsadmin"
 
-gitlab_webhook_url="http://$jenkins_user:$jenkins_password@$jenkins_elb$gitlab_trigger_job_url"
-gitlab_jazz_ui_webhook_url="http://$jenkins_user:$jenkins_password@$jenkins_elb$jazz_ui_trigger_job_url"
+if [ ! -d ./jenkins-tmp ] ; then
+    mkdir ./jenkins-tmp
+    wget -O ./jenkins-tmp/jenkins-cli.jar http://$jenkins_elb/jnlpJars/jenkins-cli.jar
+fi
+
+alias jcli="java -jar ./jenkins-tmp/jenkins-cli.jar -auth $jenkins_user:$jenkins_password -s  http://$jenkins_elb"
+
+function get_jenkinsuser_apitoken() {
```

Comment:
Is there any reason why we can’t run `scmpush.sh` alongside all the other `jenkins-cli.jar` scripts in our Chef cookbook, rather than Terraform, where we are forced to duplicate a lot of the same patterns?